teams/test-tools.toml

```diff
@@ -0,0 +1,22 @@
+name = "test-tools"
```

Comment:
> Also want to call out that in this proposal I opted to adjust the name of the team to be Test tools team. I don't feel too strongly, and am unsure if this is even permissible since we had a different name in the RFC. However, the more I looked at words in text, file names, etc. that had "Test" and "Testing" the more it felt like a "testing 1-2-3" or even a potential typo.

My only nit with the name is it makes it sound like we are focusing on `cargo test`, `cargo bench`, etc and not on libtest.

Reply:
To add to naming being hard is I saw some confusion over people thinking we were focusing on testing of rustc...

While not ideal, I'm fine with "test" and "testing".  I'm also fine with emphasizing the user experience in the name

Reply:
T-testing feels like it captures the scope better, but I don't really care. I also don't know if I think that the possible confusion is a huge problem.

Reply:
For myself, I'm still not sold on "tools".  I understand that for some, it more is being used like "resources" but naming matters and can help shape thought and behavior, both of the team and those who approach it.  We'd be working against the psychological aspects of that by biasing the name (this is especially true when the membership is also biased towards "tools").